faunadb/client_test.go

```diff
@@ -941,6 +941,51 @@ func (s *ClientTestSuite) TestEvalCasefoldExpression() {
 	s.Require().Equal("\u00E5", str)
 }
 
+func (s *ClientTestSuite) TestEvalStartsWithExpression() {
+	var b bool
+
+	s.queryAndDecode(f.StartsWith("faunadb", "fauna"), &b)
+	s.Require().Equal(true, b)
+
+	s.queryAndDecode(f.StartsWith("faunadb", "F"), &b)
+	s.Require().Equal(false, b)
+
+}
+
+func (s *ClientTestSuite) TestEvalEndsWithExpression() {
+	var b bool
+
+	s.queryAndDecode(f.EndsWith("faunadb", "fauna"), &b)
+	s.Require().Equal(false, b)
+
+	s.queryAndDecode(f.EndsWith("faunadb", ""), &b)
+	s.Require().Equal(true, b)
+}
```

Comment:
Could we change to something like `	s.queryAndDecode(f.EndsWith("faunadb", "db"), &b)` ?

Reply:
I don't think this is necessarily a problem, an empty string is a valid string and an empty set is a subset of any set.

https://en.wikipedia.org/wiki/Empty_string

Reply:
I know it's not a problem, but you should at least test with a non-empty string to make sure you are covering "all" scenarios